<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminTab</class>
 <widget class="QWidget" name="AdminTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminTab</string>
  </property>
  <widget class="JNaviButton" name="Tab1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tab1</string>
   </property>
  </widget>
  <widget class="JNaviButton" name="Tab2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tab2</string>
   </property>
  </widget>
  <widget class="JNaviButton" name="Tab3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tab3</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JNaviButton</class>
   <extends>QPushButton</extends>
   <header>jnavibutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
